content/applications/sales/point_of_sale/shop/posreceipts.rst

```diff
@@ -0,0 +1,37 @@
+========
+Receipts
+========
+
+Configuration
+=============
+
+Set up receipts by going to :menuselection:`Point of Sale --> Configuration --> Point of Sale`,
+selecting a POS, and scrolling down to the :guilabel:`Bills & Receipts` section.
+
+To **customize** the **header** and **footer**, activate :guilabel:`Header & Footer`. Doing so
+enables fields to fill in with the information you want to be printed on the receipts. To **print
+receipts** automatically once the payment is registered, enable the :guilabel:`Automatic Receipt
+Printing` setting.
```

Comment:
```suggestion
To **customize** the **header** and **footer**, activate :guilabel:`Header & Footer` and
fill in both fields with the information to be printed on the receipts.
To **print receipts** automatically once the payment is registered, enable the :guilabel:`Automatic Receipt
Printing` setting.
```
I think the first part could be simplified, just a suggestion though.
I would put the second part in a new paragraph, as it's not directly related to the first part.